src/components/KanbanBoard/KanbanBoard.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import KanbanColumn from '../KanbanColumn/KanbanColumn';
+
+class KanbanBoard extends Component {
+  constructor(props) {
+    super(props)
+  }
+  render() {
+    const data = this.props.data;
```

Comment:
If you changed the attribute to `cards` in App.js you would need to update this to `this.props.cards` 🌮